lsp-julia.el

```diff
@@ -107,13 +107,12 @@ Set to nil if you want to use the globally installed versions."
   "The command to lauch the Julia Language Server."
   `(,lsp-julia-command
     ,@lsp-julia-flags
-    ,(concat "-e using LanguageServer, Sockets, SymbolServer;"
```

Comment:
The `julia-vscode` guys load all three packages, so even if this works I would prefer to stick with this.